[Wrangler] Document missing flags for dev/publish (#6174)
cloudflare/cloudflare-docs: content/workers/wrangler/commands.md

  - The path to an entry point for your Worker.
- `--name` {{<type>}}string{{</type>}}
  - Name of the Worker.
- `--no-bundle` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}

>   - Skip Wrangler's build steps and show a preview of the script without modification

Author: ```suggestion
  - Skip Wrangler's build steps and show a preview of the script without modification. Particularly useful when using custom builds.
```

  - Host to forward requests to, defaults to the zone of project.
- `--local-protocol` {{<type>}}"http"|"https"{{</type>}} {{<prop-meta>}}(default: http){{</prop-meta>}}
  - Protocol to listen to requests on.
- `--local-upstream` {{<type>}}string{{</type>}}

>   - Host to act as origin in local mode, defaults to dev.host or route

Author: ```suggestion
  - Host to act as origin in local mode, defaults to dev.host or route.
```

>   - Host to act as origin in local mode, defaults to dev.host or route.

Reviewer: ```suggestion
  - Host to act as origin in local mode, defaults to `dev.host` or route.
```

- `--assets` {{<type>}}string{{</type>}}

>   - Root folder of static assets to be served. Unlike `--site`, `--assets` doesn't require a Worker script to serve your assets.

Reviewer: ```suggestion
  - Root folder of static assets to be served. Unlike `--site`, `--assets` does not require a Worker script to serve your assets.
```

>   - For example: `wrangler dev --name personal_blog --assets dist/ --latest`

Author: ```suggestion
  - Use in combination with `--name` and `--latest` for basic static file hosting. For example: `wrangler dev --name personal_blog --assets dist/ --latest`.
```

{{</Aside>}}

- `--experimental-local` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}

>   - Run the preview of the Worker directly on your local machine using the [open source Cloudflare Workers runtime](https://github.com/cloudflare/workerd)

Author: ```suggestion
  - Run the preview of the Worker directly on your local machine using the [open source Cloudflare Workers runtime](https://github.com/cloudflare/workerd).
```

- `--minify` {{<type>}}boolean{{</type>}}
  - Minify the script.
- `--node-compat` {{<type>}}boolean{{</type>}}

>   - Enable node.js compatibility

Author: ```suggestion
  - Enable node.js compatibility.
```

- `--persist` {{<type>}}boolean{{</type>}}

>   - Enable persistence for local mode, using default path: `.wrangler/state`

Author: ```suggestion
  - Enable persistence for local mode, using default path: `.wrangler/state`.
```

- `--persist-to` {{<type>}}string{{</type>}}
  - Specify directory to use for local persistence. Setting this flag implicitly enables `--persist`.
- `--test-scheduled` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
  - Exposes a `/__scheduled` fetch route which will trigger a scheduled event (cron trigger) for testing during development. To simulate different cron patterns, a `cron` query parameter can be passed in: `/__scheduled?cron=*+*+*+*+*`.
- `--log-level` {{<type>}}"debug"|"info"|"log"|"warn"|"error"|"none"{{</type>}} {{<prop-meta>}}(default: log){{</prop-meta>}}

>   - Specify Wrangler's logging level

Author: ```suggestion
  - Specify Wrangler's logging level.
```

  - The path to an entry point for your Worker.
- `--name` {{<type>}}string{{</type>}}
  - Name of the Worker.
- `--no-bundle` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}

>   - Skip Wrangler's build steps and directly publish script without modification

Author: ```suggestion
  - Skip Wrangler's build steps and directly publish script without modification. Particularly useful when using custom builds.
```

  - Flags to use for compatibility checks.
- `--latest` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: true){{</prop-meta>}}
  - Use the latest version of the Workers runtime.
- `--assets` {{<type>}}string{{</type>}}

>   - Root folder of static assets to be served. Unlike `--site`, `--assets` doesn't require a Worker script to serve your assets.

Reviewer: ```suggestion
  - Root folder of static assets to be served. Unlike `--site`, `--assets` does not require a Worker script to serve your assets.
```

>   - For example: `wrangler publish --name personal_blog --assets dist/ --latest`

Author: ```suggestion
  - Use in combination with `--name` and `--latest` for basic static file hosting. For example: `wrangler publish --name personal_blog --assets dist/ --latest`.
```

  - Path to a custom `tsconfig.json` file.
- `--minify` {{<type>}}boolean{{</type>}}
  - Minify the bundled script before publishing.
- `--node-compat` {{<type>}}boolean{{</type>}}

>   - Enable node.js compatibility

Author: ```suggestion
  - Enable node.js compatibility.
```
